### ISSUE TITLE
More resources for New Learners

### ISSUE THREAD

**Issue by SmashMineGame** [ForrestKnight/open-source-cs#57]
Assuming that a lot of people who look at this aren't doing a lot of programming and CS work yet, it's unlikely that they know how to use git and/or navigate github. For example, someone not familiar with github wouldn't necessary know to check the issues tab if they need some help. Maybe there could be some PDF or something they can read that can help them get started with those things? 

The same could theoretically be said for texteditors/IDEs. The courses tend to explain how to set one up for the course, but there's a lot of conflict between courses on which ones to use. Again, I think it might be helpful for people to get more information on that.

Having links to different programming communities (like discord servers) would also be helpful for a lot of people in my opinion. If you disagree, please leave a reply and let's see if we can come to an agreement!

**Comment by TruePadawan** [comment]
### **Coding Discord Servers**
[The Coding Den](https://discord.com/invite/code)
[The Programmer's Hangout](https://discord.com/invite/programming)
[Commit Your Code](https://discord.gg/r4S25ZeJ5m)

**Comment by therealironjaw** [comment]
True, but if you're made it that far, a little googling wouldn't hurt either. I just recently googled "what does fork mean in Github" after watching one for Forrest's yt videos. Maybe a FAQ sections with all the answers and/or recommendations. I'm new to GitHub as well
